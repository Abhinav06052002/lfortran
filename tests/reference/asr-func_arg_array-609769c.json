{
    "basename": "asr-func_arg_array-609769c",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/func_arg_array.f90",
    "infile_hash": "585115ceaab8814376b2532bf714e97b7bfe97787c158d257d79b778",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-func_arg_array-609769c.stderr",
    "stderr_hash": "0c56851da06e4cc6265f4f82e4d1869e85bd2016fe3705abdaac467f",
    "returncode": 2
}