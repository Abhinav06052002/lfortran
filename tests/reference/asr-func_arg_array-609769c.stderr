semantic error: Array passed into function has less elements than function expects.
 --> tests/errors/func_arg_array.f90:4:14
  |
4 |     i = temp(myArray)
  |              ^^^^^^^ 
